### ISSUE TITLE
'Album' terminology

### ISSUE THREAD

**Issue by mikeapp** [ProjectMirador/mirador-design#10]
While I understand that 'Manifest' is perhaps unclear to users, it might be worth a broader discussion in the community about how best to address this.   We've seen some terms cause issues (e.g., `attribution` in the museum domain https://github.com/IIIF/api/issues/1287 ).

**Comment by blalbrit** [comment]
I would like to register a +1 for this - for a number of user communities, 'Album' will be as confusing as 'Manifest' and a broader discussion would be appreciated.

**Comment by aeschylus** [comment]
#11 related

**Comment by regisrob** [comment]
+1
I understand the rationale of getting rid of the IIIF-specific vocabulary and choosing a generic wording for the UI, but of course I agree that "Album" will not be appropriate for many implementers... I doubt we could find a term that will satisfy everyone (is it an Item? an Document? or something more specific like a Book or a Manuscript?). It may be a non-issue, because at the end the term "Album" will not be as present in the "real" Mirador application as it is in the mockups, and it may not be a problem for implementers if it is entirely configurable and translatable.

**Comment by rsinghal** [comment]
Agreed.  It seemed like perhaps some of the mockups would have 'Album' displayed to the user.  I wonder if this is part of the configuration and there is a default word used but a developer can choose to override that with a term that better reflects their needs.

**Comment by ggeisler** [comment]
Thanks all for bringing this up. We think (at least with what we've designed so far) we won't need to actually use the terms "album" or "manifest" at all in the UI. In other words, most of the occurrences in the current mockups would be replaced with real data (e.g. "Album title" on a catalog card would be the actual manifest label/title) or can be eliminated entirely (the "Album layout" choice in the Catalog tools menu could be re-labeled as "Catalog layout" or just "Layout").

So we'll keep this in mind as we continue the design, but hopefully it's a non-issue and we can find ways to avoid using the term in the UI entirely.

**Comment by emmastanford** [comment]
If the term "album" won't be visible to users, what was the motivation behind switching from "manifest" to "album"? Changing terminology might confuse developers as well as end users.

Will "items" appear in the UI? (In the collection mockup, each album card lists the number of items (i.e., canvases) within that album.) "Item" is a confusing word for it in my opinion, because to me it suggests a greater degree of separation between items than there actually is between, for example, pages in a book.

**Comment by ggeisler** [comment]
@emmastanford:

> If the term "album" won't be visible to users, what was the motivation behind switching from "manifest" to "album"? Changing terminology might confuse developers as well as end users.

It seems that changing terminology has already confused developers but our motivation was -- since we are designing primarily for end-users who are **not** developers -- to come up with terminology that was less technical and developer-centric. As Mirador becomes more widely used I think it's safe to say that many end-users will not know what IIIF or IIIF manifests are (or should need to know).

> Will "items" appear in the UI? (In the collection mockup, each album card lists the number of items (i.e., canvases) within that album.) "Item" is a confusing word for it in my opinion, because to me it suggests a greater degree of separation between items than there actually is between, for example, pages in a book.

Do you have a suggestion for something other than "items"?

It's possible that for both manifest/album and object/item we can make those terms configurable and perhaps influenced by hints in the manifest about the type of objects being displayed (so both "items" and "pages" might be used within a given Mirador installation, depending on what is being shown in the catalog or workspace). It feels a bit too early in the design process to worry about that to me, but if you or others have suggestions about how to handle the terminology issues, it would definitely be helpful to hear them.

**Comment by ggeisler** [comment]
We were using "Album" for a manifest representing an object, not a collection. 

We've heard plenty of objection to "album" so I'm not sure hearing more of that feedback is constructive. It would be more helpful to hear ideas about how we deal with the terminology issue, as I asked in my previous comment in this thread.

**Comment by azaroth42** [comment]
Collection - Collection
Manifest - Item
Canvas - View

**Comment by sdellis** [comment]
This came up at a review with Princeton devs as well. Default labels should probably come out of some A/B usability testing, but we think this is something that should be configurable depending on the context. So, if I know I'm only dealing with books, I can relabel it in a way that makes sense to my users. 

Example:  
Collection - Shelf 
Manifest - Book
Canvas - Page

Making it configurable will allow us to easily change the defaults based on testing.
